TestSamples/SwiftUITestSample/Maestro/corruptEnvelope.yaml

```diff
@@ -2,6 +2,8 @@ appId: io.sentry.tests.SwiftUITestSample
 ---
 - launchApp
 - tapOn: "Write Corrupted Envelope"
+# The close here ensures the next corrupted envelope
+# will be present on the next app launch
 - tapOn: "Close SDK"
 - tapOn: "Write Corrupted Envelope"
```

Comment:
`l`: I guess with this the purpose is to stop the app while writing the corrupted envelope. If so, a small comment would be nice.